examples/debian-binary/flake.lock

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
Can you delete the `flake-lock` file? We don't keep the flake lock for examples.